### DISCUSSION TITLE
Best way to share posts source files?

### DISCUSSION THREAD

**Discussion by warpdesign** [GetPublii/Publii#1362] [Help]
I'd like to share the source files of my Publii web sites so that I can work on them from different computers: I'm thinking of having a git repository for that but I'm not sure which directory I should put under version control.

Is Publii using only files or some database to store posts/tags?

Note: I want to share the sources so that I can edit them, not the generated html files.

**Comment by candidexmedia** [comment]
## Version Control
> I'm not sure which directory I should put under version control

I'd recommend tracking the input folder: so [[Publii sites directory](https://getpublii.com/docs/publii-global-settings.html#fileslocation)] > [site folder] > input

## Readings on cloud sync

I'd give this article from the docs a look: https://getpublii.com/docs/publii-on-multiple-computers-via-dropbox.html

Blog version: https://getpublii.com/blog/managing-static-website-on-multiple-computers.html

You can test this with the cloud storage of your choice. For syncing between computers, you'll want to include the entire [Publii Sites directory](https://getpublii.com/docs/publii-global-settings.html#fileslocation).

## Database

> Is Publii using only files or some database to store posts/tags?

Publii stores all posts, tags info, author info, etc. in the `db.sqlite` file in the `input` folder of your site's folder. You can even open and edit that database file in something like [SQLiteStudio](https://sqlitestudio.pl/) or [DB Browser for SQLite](https://sqlitebrowser.org/).

Here's an old forum post about the db (in the context of multiple users editing a site): https://forum.getpublii.com/topic/publii-dropbox-and-syncing-the-db-sqlite-file/
